### ISSUE TITLE
Can't connect to localhost

### ISSUE THREAD

**Issue by felipeZ** [MD-Studio/cerise-mdstudio-gt#2]
Starting a job like,
```
import cerise_client.service as cc
import os

# Create a new service for user myuser, with given cluster credentials
env = os.environ
srv = cc.require_managed_service(
        'cerise-mdstudio-gt-myuser', 29593,
        'cerise-mdstudio-gt:test',
        os.environ['CERISE_GT_USERNAME'],
        os.environ['CERISE_GT_PASSWORD'])
cc.start_managed_service(srv)
```
results in the following error

```
Traceback (most recent call last):
  File "gt_run_gromacs.py", line 22, in <module>
    job = srv.create_job('decompose_include')
  File "/home/felipe/WorkbenchPython/MDStudio/venv/local/lib/python2.7/site-packages/cerise_client/service.py", line 280, in create_job
    self._create_input_dir(job_name)
  File "/home/felipe/WorkbenchPython/MDStudio/venv/local/lib/python2.7/site-packages/cerise_client/service.py", line 420, in _create_input_dir
    r = requests.request('MKCOL', self._input_dir(job_name) + '/')
  File "/home/felipe/WorkbenchPython/MDStudio/venv/local/lib/python2.7/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/felipe/WorkbenchPython/MDStudio/venv/local/lib/python2.7/site-packages/requests/sessions.py", line 508, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/felipe/WorkbenchPython/MDStudio/venv/local/lib/python2.7/site-packages/requests/sessions.py", line 618, in send
    r = adapter.send(request, **kwargs)
  File "/home/felipe/WorkbenchPython/MDStudio/venv/local/lib/python2.7/site-packages/requests/adapters.py", line 508, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=29593): Max retries exceeded with url: /files/input/decompose_include/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fcd354e2510>: Failed to establish a new connection: [Errno 111] Connection refused',))

```

**Comment by LourensVeen** [comment]
Hmm, interesting. That suggests that the WebDAV is not running, or that there's some connection/network issue. The GT specialisation is a bit stale at the moment, so perhaps it just needs to be updated to work with the latest Cerise base image. Let's have a look on Wednesday.

**Comment by LourensVeen** [comment]
The configuration file is definitely stale, which doesn't explain why it can't connect to the WebDAV though.

**Comment by LourensVeen** [comment]
Ah, the configuration had been updated, but there's a typo resulting in invalid YAML. Cerise should print an error on stdout in that case, so you can get to it via `docker log`. Or it can use the normal log file at least if the problem is in the API config, because it can still get the logging configuration from the main config.

**Comment by LourensVeen** [comment]
Log output has been added to Cerise, it ends up in /var/log/cerise/cerise_backend.err in the standard Docker configuration.

**Comment by LourensVeen** [comment]
Fixed by 9bc2a44
